tests/v3_api/test_nfs.py

```diff
@@ -0,0 +1,213 @@
+import pytest
+from .common import *  # NOQA
+
+namespace = {"p_client": None,
+             "ns": None,
+             "cluster": None,
+             "project": None,
+             "pv": None,
+             "pvc": None}
+
+PV_SERVER = os.environ.get("PV_SERVER", "")
+
+# the name of the file and its content that will be saved to the nfs
+test_file_name = "test-wl-1.txt"
+test_content = ""
+
+
+def write_content_to_file(pod):
+    cmd_write = "/bin/bash -c 'echo {1} > var/nfs/{0}'".format(test_file_name,
+                                                               test_content)
+    output = kubectl_pod_exec(pod, cmd_write)
+    assert output.strip().decode('utf-8') == ""
+
+
+def validate_file_content(pod):
```

Comment:
Can you have the content and  file name (full path) as arguments to the function?